components/service/glean/src/main/java/mozilla/components/service/glean/testing/GleanTestRule.kt

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.testing
+
+import android.content.Context
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.config.Configuration
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+class GleanTestRule(
```

Comment:
nit: Maybe some kdoc would be helpful here to understand what the purpose of this is. :)

Reply:
Oh, you're more than right. Sorry I skipped over this :) Adding them right away!